tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/mappingconstructor/MappingConstructorExprTest.java

```diff
@@ -38,6 +38,8 @@ public class MappingConstructorExprTest {
     private CompileResult inferRecordResult;
     private CompileResult spreadOpFieldResult;
     private CompileResult readOnlyFieldResult;
+    private CompileResult resultWithTupleUpdateMethod;
+    private CompileResult resultWithoutTupleUpdateMethod;
```

Comment:
Can't the tests be merged to a single file, with different functions?

Reply:
Make them local variables anyway. Add comments to say what's being tested.

Reply:
Since this is a error that happen when code generation, It is needed to be in two files